lib/stateView/contract/createContractDatabaseFactory.js

```diff
@@ -0,0 +1,29 @@
+/**
+ *
+ * @param {createSVDocumentMongoDbRepository} createSVDocumentRepository
+ * @returns {function(SVContract): Promise<*[]>}
```

Comment:
It returns `createContractDatabase`. You can create `@type` for it. Take a look at another factory as an example.